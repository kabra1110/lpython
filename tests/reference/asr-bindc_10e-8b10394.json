{
    "basename": "asr-bindc_10e-8b10394",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/bindc_10e.py",
    "infile_hash": "36d1b5d366716d6a601db544fe8ff32aba76518181e71a98a4e6500c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-bindc_10e-8b10394.stderr",
    "stderr_hash": "7686dd8c9b718548d301e28d39cd15a85b00030172c5658b38a75af5",
    "returncode": 2
}