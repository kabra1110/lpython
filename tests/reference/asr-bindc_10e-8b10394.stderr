semantic error: The struct in c_p_pointer must be C interoperable
  --> tests/errors/bindc_10e.py:23:38
   |
23 |     p2: Pointer[S] = c_p_pointer(p1, S)
   |                                      ^ not C interoperable
   |
15 |    class S:
   |    ~~~~~~~~...
...
   |
17 |        b: i64
   | ...~~~~~~~~~~ help: add the @ccallable decorator to this struct to make it C interoperable
